lib/tessel/setup-3g.js

```diff
@@ -0,0 +1,119 @@
+var Tessel = require('./tessel');
+var log = require('../log');
+
+Tessel.prototype.setup3g = function(options) {
+
+  var apn = options.apn;
+  var dialnumber = options.dialnumber;
+  var username = options.username;
+  var password = options.password;
+  var config = options.config;
+  var on = options.on;
+  var off = options.off;
+  var status = options.status;
+
+  if (config) {
+	return this.simpleExec(['uci', 'get', 'network.wan']).then(() => {
+		// If dongle already exists remove it and store new configuration
+		return this.simpleExec(['uci', 'delete', 'network.wan']).then(() => {
+			return this.configure3gDongle(apn, dialnumber, username, password);
+		});
+	}).catch(() => {
+		return this.configure3gDongle(apn. dialnumber, username, password);
```

Comment:
There appears to be a typo here. 

```suggestion
		return this.configure3gDongle(apn, dialnumber, username, password);
```